src/tree-node/actions.test.js

```diff
@@ -15,12 +15,12 @@ test('renders actions', t => {
     }
   ]
 
-  const wrapper = toJson(shallow(<Actions actions={actions} />))
+  const wrapper = toJson(shallow(<Actions actions={actions} id="snapshot" />))
```

Comment:
What's this `id` used for? `Actions` is not a component by itself, just a renderer for an array of actions. It should not have any props of its own unless we need it for something...?

Reply:
It's still a required prop so causes a warning in tests unless supplied.

Reply:
Warning: Failed prop type: The prop `id` is marked as required in `Actions`, but its value is `undefined`.
    in Actions